compatibility/sandbox-migration/src/com/daml/KeyTransfer.scala

```diff
@@ -77,8 +77,8 @@ final class KeyTransfer(
       None,
     )
 
-  override def execute(packageId: String, config: Config.Test)(
-      implicit ec: ExecutionContext,
+  override def execute(packageId: String, config: Config.Test)(implicit
```

Comment:
This hides the `implicit` qualifier a bit. It looks nicer though. And more importantly it should reduce diff noise.